www/src/pages/en/folder-structure.mdx

```diff
@@ -5,224 +5,233 @@ layout: ../../layouts/docs.astro
 lang: en
 ---
 
+import Form from "../../components/docs/folderStructureForm.astro";
+import Diagram from "../../components/docs/folderStructureDiagram.astro";
+
+Please select your packages to see the folder structure of a newly scaffolded app with those selections.
+
+<Form />
+
 The following is the folder structure of a newly scaffolded T3 App, with all options selected.
```

Comment:
This sentence needs to be changed now as it's showing the folders for the selected options, not for all options